webpack.config.js

```diff
@@ -0,0 +1,24 @@
+var path = require('path');
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+let config = {
+	entry: './src/client/index.js',
+	output: {
+		path: path.join(__dirname, '/build'),
+		filename: 'index_bundle.js',
+		publicPath: '/'
```

Comment:
path: creates /build directory
filename: outputs compiled code to this filename
publicPath: the path where your asset will be hosted (localhost:3000/index.js)